### DISCUSSION TITLE
Dialog for users to toggle automaticallyChecksForUpdates and automaticallyDownloadsUpdates?

### DISCUSSION THREAD

**Discussion by chigkim** [sparkle-project/Sparkle#2496] [Q&A] [answered]
It seems like users can toggle automaticallyChecksForUpdates when they run the app for the second time, and automaticallyDownloadsUpdates when there's an update alert.
Is there a way to present those checkboxes in the dialog when it comes up when you run checkForUpdates() even if there's no update?
Or, should I make my own UI for users to toggle those?

**Comment by zorgiepoo** [comment, marked as answer]
> It seems like users can toggle automaticallyChecksForUpdates when they run the app for the second time

This refers to the permission prompt, which allows users to toggle automaticallyDownloadsUpdates too

> Is there a way to present those checkboxes in the dialog when it comes up when you run checkForUpdates() even if there's no update? Or, should I make my own UI for users to toggle those?

You would need to [add user settings](https://sparkle-project.org/documentation/preferences-ui/) (so yes at adding your own UI for users to toggle).

**Comment by chigkim** [comment]
Thanks!
